tests/rptest/services/redpanda.py

```diff
@@ -1159,12 +1159,22 @@ def start_rp():
         if not expect_fail:
             self._started.append(node)
 
-    def start_node_with_rpk(self, node, additional_args=""):
+    def start_node_with_rpk(self, node, additional_args="", clean_node=True):
         """
         Start a single instance of redpanda using rpk. similar to start_node, 
         this function will not return until redpanda appears to have started 
         successfully.
         """
+        try:
+            if clean_node:
+                self.clean_node(node, preserve_current_install=True)
+            else:
+                self.logger.debug("%s: skip cleaning node" %
+                                  self.who_am_i(node))
+        except Exception:
```

Comment:
No need to catch and log the exception here.  When there's an exception in test, it already gets logged.